### ISSUE TITLE
Question about BeiT3 pre-training

### ISSUE THREAD

**Issue by lzy-ps** [microsoft/unilm#1225]
Recently, i'v been reading the BeiT3 paper which leaves me a deep impression.However, there's some details i don't understand. In the paper, each batch contains 2048 images, 2048 texts and 2048 image-text pairs. How do you choose the ffn ? I wonder if it's like the input before ffn is divide into 3 parts and diffenrent part belongs different expert. Moreover, what's the weight of three loss(Maked image model loss , masked language model loss and masked vision-language model loss)?

**Comment by lzy-ps** [comment]
And i would appreciate it if you could provide the pretraing code of BeiT3
